tests/ieee802154_submac/main.c

```diff
@@ -93,18 +204,133 @@ void _print_addr(uint8_t *addr, size_t addr_len)
         printf("%02x", (unsigned)addr[i]);
     }
 }
-static event_t _netdev_ev = { .handler = _netdev_isr_handler };
 
-void recv(netdev_t *dev)
+static inline int _dehex(char c, int default_)
+{
+    if ('0' <= c && c <= '9') {
+        return c - '0';
+    }
+    else if ('A' <= c && c <= 'F') {
+        return c - 'A' + 10;
+    }
+    else if ('a' <= c && c <= 'f') {
+        return c - 'a' + 10;
+    }
+    else {
+        return default_;
+    }
+}
```

Comment:
Can't you simply use `l2util` here and below?

Reply:
I guess so. But I simply copied this one from the other tests

Reply:
True saw afterwards it was just moved around